### DISCUSSION TITLE
Explorer won't change? Would appreciate any tips

### DISCUSSION THREAD

**Discussion by lore-sun** [valinet/ExplorerPatcher#4002] [Q&A]
My apologies if this is both a newb issue and or it's been addressed previously..

Fresh windows 11 24H2 (26100.2454)

Trying to enable 3rd party theme. I disabled Defender, ran the powershell command as admin, downloaded & ran exe, have selected the explorer UI tweaks I want and restarted but none of them take effect. (UltraUXthemepatcher is patched, OldnewExplorer installed if that's relevant).

Is there something I am obliviously to or must do?

Appreciate the assistance and thanks,

leo

**Comment by lore-sun** [comment]
🎲🎲
Would really appreciate some tips; even if it's simply to say this SHOULD be working or if this is a known incompatibility with 24H2 for example. Cheers

**Comment by pyrates999** [comment]
Please show the settings that you've changed.

**Comment by lore-sun** [comment]
![Screenshot 2024-12-05 133645](https://github.com/user-attachments/assets/a4a63f31-a0d8-49c3-ac0c-975d4beb5f57)

**Comment by pyrates999** [comment]
You enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Reply by lore-sun** [reply]
Appreciate the response, but that made absolutely no difference... It still looks identical. For reference the only way so far that has actually entirely removed the ribbon, search & address bar has been to enable Win 7 Command Bar. However this also totally removes the titlebar header and makes it aero/glassy which is pretty undesirable. 

![Screenshot 2024-12-06 090343](https://github.com/user-attachments/assets/8c440d15-5168-4ea8-8e0c-e17426a3effa)

**Reply by pyrates999** [reply]
You have a custom skin or theme enabled.  Try disabling that.
